deployments/liqo/templates/liqo-auth-deployment.yaml

```diff
@@ -2,6 +2,9 @@
 {{- $authConfig := (merge (dict "name" "auth" "module" "discovery" "containerName" "cert-creator") .) -}}
 {{- $awsConfig := (merge (dict "name" "aws-config" "module" "aws-config") .) -}}
 
+{{- $clusterIdConfig := (merge (dict "name" "clusterid-configmap" "module" "clusterid-configmap") .) -}}
+{{- $clusterIdConfigName := include "liqo.prefixedName" $clusterIdConfig -}}
```

Comment:
Would define the name in the helpers.tpl file, to avoid duplicating this in all files.